app/models/item.rb

```diff
@@ -1,7 +1,23 @@
 class Item < ApplicationRecord
+  include Rails.application.routes.url_helpers
+
+  after_initialize :set_defaults  
+  has_one_attached :image
+
   has_many :shipment_items
   has_many :shipments, through: :shipment_items
   has_many :order_items
   has_many :orders, through: :order_items
   validates :name, :price, presence: true
+  validates :image, presence: true
+  enum status: %w(Active Inactive)
+  enum item_type: %w(Bread Pastries Donuts Bagels)
```

Comment:
I didn't validate this yet because it will break most of the tests, will do in the future